### DISCUSSION TITLE
Can't install from marketplace

### DISCUSSION THREAD

**Discussion by bodinsamuel** [renovatebot/renovate#22856] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

Hey,

The installation from marketplace is broken, not sure why. 

### Relevant debug logs

- Go to https://github.com/marketplace/renovate
- Select free plan at the bottom
- It open the confirmation page 
- The button is not clickable (cf screenshot)

<img width="1092" alt="Screenshot 2023-06-18 at 14 49 09" src="https://github.com/renovatebot/renovate/assets/1637651/fc75dcc9-b98a-437e-ad1d-6938a8878e62">

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment, marked as answer]
1. It could be because your organization pays by invoice instead of credit card. If so, GitHub doesn't allow to buy through the Marketplace
2. It doesn't matter - you can install directly using https://github.com/apps/renovate

**Comment by bodinsamuel** [comment]
1. Weird that my billing choice would impact a free app and that github do not explains it, but you probably now more than me 👍🏻 
2. Yes that's almost what I end up doing, except via mend.io 

As long as it doesn't impact many users, should be good 👍🏻 
Thanks
